src/constants.ts

```diff
@@ -1,3 +1,6 @@
 export const NOISE_MSG_MAX_LENGTH_BYTES = 65535;
 export const NOISE_MSG_MAX_LENGTH_BYTES_WITHOUT_TAG = NOISE_MSG_MAX_LENGTH_BYTES - 16;
 
+export const DUMP_SESSION_KEYS = false;
```

Comment:
```suggestion
export const DUMP_SESSION_KEYS = process.env.NOISE_DUMP_KEYS;
```

Reply:
You already need DEBUG=libp2p:session as an enviroment variable to show session keys. This  const made sense so you won't accidentally leave an enviroment variable active and show them. So in my opinion either we don't need this variable or leave as a constant.

Reply:
oh don't do that, libp2p:session is meant for logs called js-libp2p-session. Usually people will log libp2p stuff with DEBUG=libp2p:* which would log keys by accident and we should avoid that

Reply:
Definitely should change DEBUG to something more specific to this, eg. noise:session-keys?

Reply:
it should stay libp2p:noise, but logging of keys should depend on additional env variable